app/models/manageiq/providers/vmware/infra_manager/vapp.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Vmware::InfraManager::Vapp < Vapp
```

Comment:
maybe we should call this `VirtualApp` or even the more verbose `VirtualApplication` for clarity?